recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ outputs:
       run:
         - python
         - setuptools
+        - toml
```

Comment:
`pip check` did not catch that :-/

Reply:
It's in `extras_require`, you normally install it with `setuptools_scm[toml]` so I wouldn't expect it to.

Reply:
Ah. But then we should not add it to the package, we decided to leave extras to the user. Anyways, this is small enough that I'm OK keeping it there.